bech32/app/Main.hs

```diff
@@ -100,19 +109,20 @@ parse = customExecParser (prefs showHelpOnEmpty) parser
         ]
 
     cmd :: Parser Cmd
-    cmd = Cmd <$> optional hrpArgument
+    cmd = (RunCmd <$> optional hrpArgument) <|> (VersionCmd <$ versionSwitch)
```

Comment:
This looks a bit odd.
```
bech32 [PREFIX | (-v|--version)]
```
Can you hide the `--version` flag from the usage text, but still have it shown in the options list?
I _think_ the modifier is [Options.Applicative.hidden](https://hackage.haskell.org/package/optparse-applicative-0.14.2.0/docs/Options-Applicative.html#v:hidden).